watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Tasks/TaskFragment.java

```diff
@@ -193,11 +199,20 @@ public void onResponse(ArrayList<Task> tasks) {
                         hideList();
                     }
                 }
+                setSwipeFalse();
+            }
+        }, this);
+        mNetworkManager.getRequestQueue().add(taskListRequest);
+    }
 
+    public void setSwipeFalse() {
+        new CountDownTimer(1000, 1000) {
+            public void onTick(long millisUntilFinished) {}
+            public void onFinish() {
                 if (mSwipeLayout != null) mSwipeLayout.setRefreshing(false);
+                if (mNoTasks != null) mNoTasks.setRefreshing(false);
             }
-        }, mSwipeLayout);
-        mNetworkManager.getRequestQueue().add(taskListRequest);
+        }.start();
```

Comment:
I still think we should try to find an alternative to this. Really doesn't make sense to make the app appear intentionally slower than it is.